scargo/jinja/docker/Dockerfile-custom.j2

```diff
@@ -1,4 +0,0 @@
-# user can add his docker code below which will be integrated with 
```

Comment:
Not sure why you removing that, it info is for user who extending docker file, if the start with FROM, then they crash scargo docker file